release/release.go

```diff
@@ -256,6 +263,8 @@ type Module struct {
 	// The changes for the module
 	Changes ModuleChange
 
+	FileChanges []string
```

Comment:
Kind of a minor item, but I was trying to keep a JSON Schema description of this output in case other want to use this JSON description in other tooling. If you wouldn't mind adding this new field to that description: https://github.com/awslabs/aws-go-multi-module-repository-tools/blob/main/release/manifest_schema.json

Reply:
Thanks, missed that. Will update the docs